src/dev-app/theme-m3.scss

```diff
@@ -13,7 +13,9 @@ mat.$theme-legacy-inspection-api-compatibility: false;
       theme-type: $type,
       primary: matx.$m3-azure-palette,
       tertiary: matx.$m3-blue-palette,
+      use-sys-vars: true,
```

Comment:
Should we make this a bit more descriptive like `use-system-level-variables`? This isn't something users would have to write a lot so it's better to be clear than short.